lib/ae_mdw/aexn_tokens.ex

```diff
@@ -76,10 +80,13 @@ defmodule AeMdw.AexnTokens do
       state
       |> Collection.stream(@aexn_creation_table, direction, key_boundary, cursor)
       |> Stream.map(fn {aexn_type, txi_idx} ->
-        Model.aexn_contract_creation(contract_pk: pubkey) =
-          State.fetch!(state, @aexn_creation_table, {aexn_type, txi_idx})
-
-        State.fetch!(state, @aexn_table, {aexn_type, pubkey})
+        State.fetch!(state, @aexn_creation_table, {aexn_type, txi_idx})
+      end)
+      |> Stream.reject(fn Model.aexn_contract_creation(contract_pk: pubkey) ->
```

Comment:
If we ignore the contracts from the listings we're going to get a lot of complaints, what I would do instead is display `invalid: true` when rendering them